### PR TITLE
Docs: Update section about time range with copy/paste functionalities

### LINKED ISSUE #80846: Add docs for Copy-Paste Time Range keyboard shortcuts
Add docs for keyboard shortcuts implemented in this PR: https://github.com/grafana/grafana/pull/80107

### PR DESCRIPTION
**Pull request by harisrozajac**
**What is this feature?**

Adds documentation for copy/paste time range functionality implemented in #80107

Closes #80846 

Please check that:
- [ ] That the language makes sense

### CODE REVIEW

File: docs/sources/dashboards/use-dashboards/index.md [hunk 1]
```diff
@@ -190,6 +190,20 @@ This is equivalent to the **Today so far** time range preset, but it starts at 8
...
+
+##### Copy time range (t+c)
+
+<img class="no-shadow" src="/media/docs/grafana/dashboards/screenshot-copy-paste-time-range.png" max-width="900">
+
+Click the **Copy time range to clipboard** icon to copy the current time range to the clipboard. You can then paste the time range into Explore.
+
+##### Paste time range (t+v)
+
+Click the **Paste time range** icon to paste the time range from the clipboard into the current time range. Currently
```
Comment (imatwawana) [inline docs/sources/dashboards/use-dashboards/index.md:205]: Most people are familiar with the copy/paste function on a keyboard, so we don't need to be so detailed in this explanation or use as many headings.
```suggestion
Click the **Copy time range to clipboard** icon to copy the current time range to the clipboard. Then paste the time range into Explore or another dashboard.

<img class="no-shadow" src="/media/docs/grafana/dashboards/screenshot-copy-paste-time-range.png" max-width="900">

You can also use the keyboard shortcuts `t+c` and `t+v` to copy and paste respectively.
```

File: docs/sources/dashboards/use-dashboards/index.md [hunk 2]
```diff
@@ -190,6 +190,20 @@ This is equivalent to the **Today so far** time range preset, but it starts at 8
 
 Using a semi-relative time range, as time progresses, your dashboard will automatically and progressively zoom out to show more history and fewer details. At the same rate, as high data resolution decreases, historical trends over the entire time period will become more clear.
 
+#### Copy and paste time range
+
+You can copy and paste time range from a dashboard to Explore and vice versa. You can also copy and paste time range from one dashboard to another.
```
Comment (imatwawana) [inline docs/sources/dashboards/use-dashboards/index.md:195]:
```suggestion
You can copy and paste time range from a dashboard to Explore and vice versa, or from one dashboard to another.
```

**Review by imatwawana — COMMENTED** [review]
Thanks for making this update! I've made some suggestions for how to streamline this addition as it's a bit too detailed at the moment.

**Comment by harisrozajac** [comment]
@imatwawana thank you for the feedback! i've committed your suggestions, so we should be good to go 🚀

**Review by imatwawana — APPROVED** [review]
Awesome, thank you!

**Pull request merged**